modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -117,4 +119,4 @@ For the *All namespaces...* installation mode, the status resolves to *InstallSu
 +
 If it does not:
 
-.. Check the logs in any pods in the `openshift-operators` project (or other relevant namespace if *A specific namespace...* installation mode was selected) on the *Workloads → Pods* page that are reporting issues to troubleshoot further.
+** Check the logs in any pods in the `openshift-operators` project (or other relevant namespace if *A specific namespace...* installation mode was selected) on the *Workloads → Pods* page that are reporting issues to troubleshoot further.
```

Comment:
```suggestion
** Check the logs in any pods in the `openshift-operators` project (or other relevant namespace if you selected the `SingleNamespace` install  mode) on the *Workloads → Pods* page that are reporting issues to troubleshoot further.
```

Reply:
Leaving as-is because the web console doesn't refer to it as `SingleNamespace`.